### PR TITLE
TTS update

### PR DESCRIPTION
**Pull request by vvvv-vvvv**
## About The Pull Request

Updated the TTS stuff, mostly code improvements but also there is a lowpass filter on gas masks now

## Changelog

:cl:
qol: Gas masks now muffle your voice with TTS.
/:cl:

### CODE REVIEW

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -19,6 +19,7 @@
 	var/breathy = TRUE
 	///This covers most of the screen
 	var/hearing_range = 5
+	voice_filter = "lowpass=f=750,volume=2"
```
Comment (TiviPlus) [inline code/modules/clothing/masks/gasmask.dm:22]: new vars after old ones

**Review by TiviPlus — COMMENTED** [review]
rest ok i guess

**Review by TiviPlus — APPROVED** [review]

**Comment by Lumipharon** [comment]
Do we actually want gas mask effects? Because that'll be applied to everyone, basically.

**Comment by FosterGrime-Heymynameslouie** [comment]
I do think we want gasmasks muffling your voice it sounds cool

**Comment by carlarctg** [comment]
> Do we actually want gas mask effects? Because that'll be applied to everyone, basically.

Let people disable it if they want?

**Comment by Sirsnek15** [comment]
can they be disabled by default they're like super buzzy in the example provided

**Comment by Sirsnek15** [comment]
Also, what qualifies as gasmask? Ski mask has all the benefits but should not have the sound change

**Pull request merged**